package-lock.json

```diff
@@ -2010,7 +2002,7 @@
 		"@wordpress/block-library": {
 			"version": "file:packages/block-library",
 			"requires": {
-				"@babel/runtime": "^7.0.0-beta.52",
+				"@babel/runtime-corejs2": "7.0.0-beta.56",
```

Comment:
I'm going to revert this change and restore lines 707-714 above.

Let's have this PR only updatesha512 hash for ``

All the Babel deps have/are being updated for the 7.0.0 release in #9171

Reply:
The change was one which should have been made to a previous dependency update and is present in master (#8911). I think it should have remained.